src/routes/v0.ts

```diff
@@ -42,6 +42,10 @@ function translateIncentives(incentives: IRAIncentive[]): WebsiteIncentive[] {
       'https://www.rewiringamerica.org',
       '',
     ),
+    // TODO: remove when PEP is migrated to v1 API
+    more_info_url_internal:
+      incentive.more_info_url?.en ||
+      tr(PROGRAMS[incentive.program as keyof Programs].name, 'en'),
```

Comment:
`incentive.more_info_url` should also be passed to `tr`, right? (I know this is supposed to be temporary but I could imagine the new pages being translated into Spanish before we're ready to move PEP off of v0)

And if you make more_info_url required, you can remove the fallback. (Also the fallback isn't a URL?)

Reply:
PEP doesn't support translations so I didn't think it would matter since we would never send with `language: es` param - but will update just in case